multiversx_sdk_cli/constants.py

```diff
@@ -11,3 +11,5 @@
 
 DEFAULT_HRP = "erd"
 ADDRESS_ZERO_BECH32 = "erd1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq6gq4hu"
+
+NUMBER_OF_SHARDS = 3
```

Comment:
Indeed the `number_of_shards` is not used to compute the contract address, added it to be more explicit, but it can be deleted since the default value is already `3`. Let me know what you think.